--- build/tarballs/lua-5.1.5/src/llex.c	2009-11-23 22:58:22.000000000 +0800
+++ llex.c	2013-09-22 21:55:41.481941014 +0800
@@ -178,9 +178,10 @@
 
 static void trydecpoint (LexState *ls, SemInfo *seminfo) {
   /* format error: try to update decimal point separator */
-  struct lconv *cv = localeconv();
+  /* struct lconv *cv = localeconv();*/
   char old = ls->decpoint;
-  ls->decpoint = (cv ? cv->decimal_point[0] : '.');
+  /* ls->decpoint = (cv ? cv->decimal_point[0] : '.');*/
+  ls->decpoint = '.';
   buffreplace(ls, old, ls->decpoint);  /* try updated decimal separator */
   if (!luaO_str2d(luaZ_buffer(ls->buff), &seminfo->r)) {
     /* format error with correct decimal point: no more options */
